### PR TITLE
FSET-725 Extend Video Interview

### PR DESCRIPTION
**Pull request by benjstephenson**

### CODE REVIEW

File: launchpad-application.conf.enc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+U2FsdGVkX19VY5oszzHPSgsJfKVIhOP/Mp8dXzsuEOCs+wC9A4fDE1R8e8KCQGdL
+BQZ7KQyqtHeu4VmdXcOTepaHG0eXiidbx0dNaLywwTPm2VY0cfwTyGm5NjKgoxML
```
Comment (henricook) [inline launchpad-application.conf.enc]: This probably shouldn't be here(!)

File: test/uk/gov/hmrc/fsetlaunchpadgateway/connectors/launchpad/ManualClient.scala [hunk 2]
```diff
@@ -6,41 +6,61 @@ import play.api.Logger
...
 
-// TODO: Entirely remove the manual client after initial setup when it ceases to be useful
-class ManualClient extends UnitSpec with OneServerPerTest {
+// This is a manual test class for trying new methods against the launchpad API
+class ManualClient extends UnitSpec with OneServerPerTest {}
 
+/*
   "Testing" should {
-    "Client test" ignore {
+    "Client test" in {
```
Comment (henricook) [inline test/uk/gov/hmrc/fsetlaunchpadgateway/connectors/launchpad/ManualClient.scala]: This needs to be ignore otherwise travis is going to run it

Comment (benjstephenson) [inline test/uk/gov/hmrc/fsetlaunchpadgateway/connectors/launchpad/ManualClient.scala]: The test is commented out

Comment (henricook) [inline test/uk/gov/hmrc/fsetlaunchpadgateway/connectors/launchpad/ManualClient.scala]: Right you are :eyes:

**Pull request merged**